src/test/resources/application-test.yml

```diff
@@ -17,6 +17,9 @@ spring:
     username: non_associations
     password: non_associations
 
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
i tried to read about what this is doing and i'm still unsure how this helps with transactions and/or annotations on the entities. is this here just to match other projects?

Reply:
It might not be needed now.  I take out in next PR

Reply:
oh i'm happy to leave it if it's necessary for other things or helps align with other projects